src/uct/ib/base/ib_md.c

```diff
@@ -1464,7 +1462,7 @@ ucs_status_t uct_ib_md_open(uct_component_t *component, const char *md_name,
     uct_ib_md_t *md = NULL;
     struct ibv_device **ib_device_list, *ib_device;
     uct_ib_md_ops_entry_t *md_ops_entry;
-    int i, num_devices, ret;
+    int i, num_devices, ret, fork_init = 0;
```

Comment:
maybe no need for fork_init local var?
just set md->fork_init directly

Reply:
don't we want to call ibv_fork_init() before ibv_open_device() and all the md initialization?

Reply:
right